azurerm/resource_arm_nat_gateway.go

```diff
@@ -0,0 +1,269 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2019-07-01/network"
```

Comment:
We've updated the API version used on master so this will have to change

Reply:
Done